scripts/train_keras.py

```diff
@@ -26,13 +26,16 @@
 from keras.optimizers import SGD, RMSprop
 from keras import backend as K
 from keras.callbacks import TensorBoard
+from keras.utils import multi_gpu_model
+import tensorflow as tf
 import numpy as np
-from . import score_keras
+import score_keras
 
 from datetime import datetime
 import os
 import argparse
 import logging
+from distutils.util import strtobool
```

Comment:
Thanks for this. I used always-true flags and forgot to remove the "True" in the command above (e.g. `--use-weights` works), but your solution is much more python-friendly.